discord/activity.py

```diff
@@ -408,16 +420,20 @@ def to_dict(self) -> Dict[str, Any]:
             'type': ActivityType.playing.value,
             'name': str(self.name),
             'timestamps': timestamps,
+            'platform': str(self.platform) if self.platform else None,
+            'assets': self.assets,
         }
 
     def __eq__(self, other: object) -> bool:
-        return isinstance(other, Game) and other.name == self.name
+        return (isinstance(other, Game)
+                and other.name == self.name
+                and other.platform == self.platform)
```

Comment:
I don't think `platform` should be part of the `__eq__` or `__hash__` though I guess it's up to debate.